README.md

```diff
@@ -160,8 +161,17 @@ cargo {
 
 A list of Android targets to build with Cargo; required.
 
-Valid targets are `arm`, `arm64`, `x86`, `x86_64` (Android), and `'linux-x86-64'`, `'darwin'`,
-`'win32-x86-64-gnu'`, and `'win32-x86-64-msvc'` (Desktop).
+Valid targets for **Android** are:
+
+>`'arm'`, `'arm64'`, `'x86'`, `'x86_64'`
```

Comment:
This is an improvement, but let's make it way easier to copy-paste: maybe just a block with each entry on a line, or a list?